### ISSUE TITLE
FW crashing without bootloader ?

### ISSUE THREAD

**Issue by sdorre** [endian-albin/pinetime-hypnos#90]
Is it just me or the FW is crashing regularly when compiled and flashed without bootloader support ?

I just touch the screen a few times and get BUS FAULT at some point. (SOmethime the first touch, sometime after a swipe or aftera few touch events ...)

```
[00:00:07.038,299] [1;31m<err> os: ***** BUS FAULT *****[0m
[00:00:07.038,299] [1;31m<err> os:   Instruction bus error[0m
[00:00:07.038,299] [1;31m<err> os: r0/a1:  0x00000000  r1/a2:  0x00005835  r2/a3:  0x2000770c[0m
[00:00:07.038,330] [1;31m<err> os: r3/a4:  0x1f118ff3 r12/ip:  0x00000000 r14/lr:  0x0002814f[0m
[00:00:07.038,330] [1;31m<err> os:  xpsr:  0x01000000[0m
[00:00:07.038,330] [1;31m<err> os: Faulting instruction address (r15/pc): 0x1f118ff2[0m
[00:00:07.038,330] [1;31m<err> os: >>> ZEPHYR FATAL ERROR 0: CPU exception on CPU 0[0m
```

**Comment by endian-albin** [comment]
I almost never use the device without bootloader support so I haven't see this. Have you also enabled logging?

**Comment by endian-albin** [comment]
It crashes for me as well (freezes) with logging disabled. It's interesting that it doesn't happen with the bootloader version.

**Comment by endian-albin** [comment]
Fixed it in 8c634b89e0dc742042f6fad8515234618ecc1fa1.
